md_api/src/main/java/com/wipro/ats/bdre/md/api/HaltStep.java

```diff
@@ -29,25 +29,26 @@
  * Created by arijit on 12/8/14.
  */
 public class HaltStep extends MetadataAPIBase {
-    public HaltStep() {
-        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
-        acbFactory.autowireBean(this);
-    }
 
     private static final Logger LOGGER = Logger.getLogger(HaltStep.class);
     private static final String[][] PARAMS_STRUCTURE = {
             {"p", "sub-process-id", "Sub Process id of the step"}
     };
 
+    @Autowired
+    private StepDAO stepDAO;
+
+    public HaltStep() {
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
+        acbFactory.autowireBean(this);
+    }
+
     /**
      * This process calls HaltStep Proc and updates instance_exec table and batch_consump_queue.
      *
      * @param params String array having sub-process-id, environment with their command line notations.
      * @return returns nothing.
      */
-    @Autowired
-    private StepDAO stepDAO;
-
     public HaltStepInfo execute(String[] params) {
 //        SqlSession s = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)